styles/Button.module.scss

```diff
@@ -0,0 +1,56 @@
+.button {
+    display: block;
+    position: relative;
+    @include z-index(zero);
+    width: max-content;
+    @include font-size(m);
+    font-weight: $weight--extrabold;
+    color: var(--color--white);
+    padding: 12px 50px 12px 20px;
+
```

Comment:
usually we avoid spaces in css files like this, I personally quite like when having it to seperate specific values like say your background properties but this would usually be flagged by over reviewers so thought I'd mention.